lib/bootloader_pvm.pm

```diff
@@ -117,10 +117,13 @@ sub prepare_pvm_installation {
 
     # Delete partition table before starting installation
     select_console('install-shell');
-    foreach my $disk (split(',', get_var('DISK_DEVICES', 'sda'))) {
-        script_run("wipefs -a /dev/$disk");
-        # For cryptlvm+activate existing scenario, create empty enrypted partition
-        create_encrypted_part("$disk") if get_var('ENCRYPT_ACTIVATE_EXISTING');
+    
+
+    my $disks = script_output('lsblk -n -p -l -o NAME -d -e 7,11');
```

Comment:
I remember issues with script_output being polluted, but I guess in the beginning of the installation we should not see them.